saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -713,14 +716,25 @@ def perform_mutation(cls, root, info, **data):
     @classmethod
     @traced_atomic_transaction()
     def update_or_create_variant_stocks(cls, variant, stocks_data, warehouses):
+        product_variant_stock_webhooks = ProductVariantStockWebhookTrigger()
         stocks = []
         for stock_data, warehouse in zip(stocks_data, warehouses):
-            stock, _ = warehouse_models.Stock.objects.get_or_create(
+            stock, is_created = warehouse_models.Stock.objects.get_or_create(
                 product_variant=variant, warehouse=warehouse
             )
+            product_variant_stock_webhooks.append_stock_data(
+                stock, is_created, stock.quantity, stock_data["quantity"]
+            )
             stock.quantity = stock_data["quantity"]
             stocks.append(stock)
         warehouse_models.Stock.objects.bulk_update(stocks, ["quantity"])
+        plugins_manager = get_plugins_manager()
```

Comment:
We should avid fetching `plugins_manager` from the database. Could we get it from context?
(`manager = info.context.plugins`)